docs/src/pages/reference/useQuery.md

```diff
@@ -117,6 +118,9 @@ const result = useQuery({
   - Optional
   - If set, the component will not re-render if any of the listed properties change.
   - If set to `['isStale']` for example, the component will not re-render when the `isStale` property changes.
+- `notifyOnChangeTracked`
```

Comment:
Maybe `notifyOnChangeTrackedProps` would be a bit more correct. And should the option be enabled by default? It could prevent a lot of re-renders out of the box.

Reply:
So, I've been thinking: I don't really like having 3 fields that tackle the same area, especially if we enable tracking per default. One possibility would be to _merge_ this new feature with `notifyOnChangeProps`:

```
notifyOnChangeProps?: Array<keyof InfiniteQueryObserverResult> | 'tracked'
```

and have it default to `'tracked'` (or a different, better name 😅 ). That way, everyone who has now already optimized manually via e.g. `notifyOnChangeProps: ['data', 'error']` would keep the same behaviour, but per default, we would track the changes, and we wouldn't need to add a new option field ....

Reply:
Sounds good, it does discard the option to specify some properties manually in addition to the tracked properties, but I'm not sure if people would needs this.

Reply:
I also don't think that's a use-case, so I'll go ahead with the proposal 👍

Reply:
here it is: ad540e2d